coderd/notifications/testdata/rendered-templates/TemplateUserRequestedOneTimePasscode-body.md.golden

```diff
@@ -0,0 +1,7 @@
+Hi Bobby,
+
+A request to reset the password for your Coder account has been made. Your one-time passcode is:
+
+**fad9020b-6562-4cdb-87f1-0486f1bea415**
```

Comment:
nit: alternative password, easier to comprehend is a combination of words from a dictionary (assuming that OTPs expire)

for example: `angry-green-swan-94`